base/options/options.html

```diff
@@ -270,7 +270,7 @@
 							<input type="text" id="vOffset" class="mySelect" style="width: 50px;" />
 							<p align="justify" style="display: inline; font-size: 12px;">
 								For Vertical Location offset. Only Numbers are allowed.
-							</p>
+							</p>K
```

Comment:
what is this 'K'

Reply:
Oh sorry, that's my fault. I create this PR by a mobile phone, and I insert "K" occasionally.

Reply:
ok will add the link in the next release, thanks for your contribution